tools/launchdev.sh

```diff
@@ -182,9 +182,9 @@ function st2start(){
     # option so we need to use screen config file per screen window
 
     # Run the st2 API server
-    echo 'Starting screen session st2-api...'
+    echo 'Starting screen session st2-api'
```

Comment:
Extract launchdev.sh cleanups into a separate PR (probably including the switch from screen to tmux in #6157.

Reply:
#6186